akvo/rsr/models/report.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+# Akvo RSR is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ..fields import ValidXMLCharField, ValidXMLTextField
+
+
+class ReportFormat(models.Model):
+    name = ValidXMLCharField(_('name'), max_length=30, unique=True)
```

Comment:
This is kind of a nit-pick, but I think this field should either be named **format**, since that's how it's used or **document_type** because that's what it is.